build_tools/cmake/build_riscv.sh

```diff
@@ -98,3 +98,9 @@ esac
 args_str=$(IFS=' ' ; echo "${args[*]}")
 "${CMAKE_BIN}" ${args_str} "${ROOT_DIR}"
 "${CMAKE_BIN}" --build "${BUILD_RISCV_DIR}" -- -k 0
+
+if [[ "${RISCV_ARCH}" == "rv64" ]]; then
+  echo "Building test deps for RISC-V"
+  echo "------------------"
```

Comment:
nit: we're inconsistent with these, but I think the intent with the `-----` was to match the length of the previous echo

```suggestion
  echo "Building test deps for RISC-V"
  echo "-----------------------------"
```

Reply:
Done.